producers/smd_producer.py

```diff
@@ -44,6 +44,35 @@ def getROIs(run):
         ret_dict['jungfrau1M'] = roi_dict
     return ret_dict
 
+def getAzIntPyFAIParams(run):
+    if isinstance(run,str):
+        run=int(run)
+    ret_dict = {}
+    if run>0:
+        az_dict = {}
+        pix_size = 176e-6
+        ai_kwargs = {'dist':1, 'poni1':960*pix_size, 'poni2':960*pix_size}
+        az_dict['ai_kwargs'] = ai_kwargs
+        az_dict['npts'] = 512
+        az_dict['int_units'] = '2th_deg'
+        az_dict['return2d'] = False
+        ret_dict['Rayonix'] = az_dict
+    return ret_dict
+
+# def getAzIntParams(run):
```

Comment:
Why does this need to get commended out?

Reply:
Yes, this should be removed and we should only keep the function we want to use for the experiment. All the definition can be copied from the smd_producer_template.py if needed. This part needs to be edited and cleaned up for each experiment.